File: baekjoon/javascript/2475 검증수.js
const input = require("fs").readFileSync("/dev/stdin").toString().trim().split(" ").map(n=>+n);
let arr = []
for(let i = 0; i<input.length ; i++){
arr.push(Math.pow(input[i],2));
}
const a = arr.reduce((a,b) => (a+b));
console.log(a%10);

File: baekjoon/dart/2475_검증수.dart
import 'dart:math';

void main(List<String> args) {
  String input = "0 4 2 5 6";
  List<String> numbers = input.split(" ");
  List<int> nums = numbers.map((e) => int.parse(e)).toList();
  List<num> arr = nums.map((num) => pow(num, 2)).toList();
  num result = arr.reduce((a, b) => (a + b));
  print(result % 10);
}
